src/main/scala/beam/agentsim/agents/vehicles/BeamVehicle.scala

```diff
@@ -396,9 +396,9 @@ class BeamVehicle(
     val mode = beamVehicleType.vehicleCategory match {
       case Bike =>
         BIKE
-      case Car | LightDutyTruck | HeavyDutyTruck if isCAV =>
+      case Car | LightDutyTruck | MediumDutyPassenger | HeavyDutyTruck if isCAV =>
```

Comment:
Somebody please verify this - I'm surprised it wasn't caught as a match miss before the 2.13 change. But I believe they explicitly worked on improving the compiler in this regard.